### PR TITLE
Fix ContentDB packages timing out by using download_file instead

### LINKED ISSUE #7447: Content store does not retrieve any packages
##### Issue type

 - Bug report
 
##### Minetest version

```
0.5.0-dev-9a1d3584
```

##### Operating system
Ubuntu 18.04 LTS

##### Summary

Here are the relevant error messages returned:
```
ERROR[Main]: https://content.minetest.net/packages/ not found (Timeout was reached) (response code 200)
ERROR[Main]: Invalid JSON download
```

- Content store tends to work fine on the first run after PC startup.
- Problem starts when MT is closed and reopened, and persists in the subsequent runs.

image

### PR DESCRIPTION
**Pull request by rubenwardy**
Fixes #7447

### CODE REVIEW

File: src/script/lua_api/l_mainmenu.cpp [hunk 1]
```diff
@@ -991,68 +991,6 @@ int ModApiMainMenu::l_get_screen_info(lua_State *L)
 	return 1;
 }
 
-int ModApiMainMenu::l_get_package_list(lua_State *L)
```
Comment (SmallJoker) [inline src/script/lua_api/l_mainmenu.cpp]: Also remove from `menu_lua_api.txt`

**Comment by paramat** [comment]
Seems to work, takes 4s for me to enter package menu.
Is it normal for this to create folders in my computer's 'tmp' folder? I have several folders named like:
`/tmp/MTTempModFolder_3018` which gradually fill up with images as i browse through the package list.
Assuming this is a cache of images why does this not appear in my (RUN IN PLACE = 1) MT folder? Probably fine just wondering.

**Comment by rubenwardy** [comment]
> It creates folders in my computer's 'tmp' folder. I have several folders named like:
/tmp/MTTempModFolder_3018 which gradually fill up with images as i browse through the package list. A new folder is added every time i run MT.

This is also the case with current master

**Comment by paramat** [comment]
:+1: not an expert on file stuff but code seems ok, i think it's important this gets merged for testing soon.

**Pull request merged**